airbyte-integrations/connectors/source-stripe/metadata.yaml

```diff
@@ -43,6 +43,12 @@ data:
         scopedImpact:
           - scopeType: stream
             impactedScopes: ["refunds"]
+      6.0.0:
```

Comment:
`customer` was set as object, and now it is changed to `string`.
This is the main reason to mark changes as breaking.
endpoint change can also affect customer incremental sync.

Reply:
shouldn't this be part of the migration guide then?

Reply:
🌶️ I'd still advocate for not flagging as breaking change since only one Cloud user actually has data for this stream

Reply:
ok, reverted.

Reply:
> shouldn't this be part of the migration guide then?

Since i removed this version from `breakingChanges` in metadata, CI fails on QA checks in migration guide.
Is there any other place I can place the release notes?